### DISCUSSION TITLE
Mermaid render error gantt chart

### DISCUSSION THREAD

**Discussion by xcojonny** [requarks/wiki#4643] [Error / Bug Report]
Hello,

I'm using mermaid charts within markdown. Standard stuff works pretty fine. Thanks for that:

```
'''mermaid
graph LR
		Trigger(Trigger Button,  optional) -->|TRIG SW IN| CAM
    CAM -.->|Ethernet| Laptop
'''
```

If i use gantt charts the chart is not rendered properly. In edit mode the ganttchart is rendered but not when i exit the edit mode. The section is then rendered as a code section. Example which does only work in edit mode using the markdown editor:

```
'''mermaid
gantt
    title A Gantt Diagram
    dateFormat  YYYY-MM-DD
    section Section
    A task           :a1, 2014-01-01, 30d
    Another task     :after a1  , 20d
    section Another
    Task in sec      :2014-01-12  , 12d
    another task      : 24d
'''    
```

Thanks in advance!!!!!

**Comment by ieaster1** [comment]
What version are you running?

Are you using the single quotes (') for your block or backticks (`)?

This block worked for me on version 2.5.219.
\```mermaid
gantt
    title A Gantt Diagram
    dateFormat  YYYY-MM-DD
    section Section
    A task           :a1, 2014-01-01, 30d
    Another task     :after a1  , 20d
    section Another
    Task in sec      :2014-01-12  , 12d
    another task      : 24d
\```

**Comment by xcojonny** [comment]
Ah. Updating to the latest version fixed the problem.
Thanks for your time and help !!!

Am 03.11.2021 um 23:16 schrieb Ian Easter ***@***.***>:

﻿

What version are you running?

Are you using the single quotes (') for your block or backticks (`)?

This block worked for me on version 2.5.219.
```mermaid
gantt
title A Gantt Diagram
dateFormat YYYY-MM-DD
section Section
A task :a1, 2014-01-01, 30d
Another task :after a1 , 20d
section Another
Task in sec :2014-01-12 , 12d
another task : 24d
```
